content/20.treatments.md

```diff
@@ -132,6 +171,31 @@ A list of current studies and their results, using carefully the information req
 
 Summarize the state of the neutralizing antibody approach.
 
+#### Interferons
+
+Interferons (IFNs) are a family of cytokines crucial to activate the first (innate) immune system response against viral infections. 
+Interferons are classified into three categories based on their receptor specificity: type I, II and III (doi.org/10.1128/mmbr.05015-11). 
+Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾 ) induce the expression of antiviral proteins which bring the viral RNA to degradation (doi.org/10.1016/j.jcv.2003.11.013). 
+Among these IFNs, IFN- 𝛽 was already found to strongly inhibit the replication of other corona viruses, such as SARS-Cov, in cell culture, while IFN-𝛼 and 𝛾 were shown to be less effective in this context (doi.org/10.1016/j.jcv.2003.11.013). 
+There are evidences that patients with higher susceptibility to develop Acute respiratory distress syndrome (ARDS) show indeed deficiency of IFN- 𝛽. 
+For instance, upon other Corona viruses infection IFN- 𝛽 expression and synthesis is impaired, so that the virus can in fact escape the innate immune response (doi.org/10.1016/j.virusres.2014.07.024). 
```

Comment:
```suggestion
For instance, upon other Corona viruses infection IFN- 𝛽 expression and synthesis is impaired, so that the virus can in fact escape the innate immune response [@doi:10.1016/j.virusres.2014.07.024]. 
```